src/lib/rocksdb/database.ml

```diff
@@ -57,21 +57,19 @@ module Batch = struct
     Rocks.write t.db batch ; result
 end
 
-let copy _t = failwith "copy: not implemented"
-
 let remove t ~(key : Bigstring.t) : unit =
   Rocks.delete ?pos:None ?len:None ?opts:None t.db key
 
+let copy t : Bigstring.t =
```

Comment:
Aren't these changes already in the base branch (https://github.com/MinaProtocol/mina/pull/15199)? Same comments apply here as there, but I don't understand how they ended up in this PR at all

Reply:
They shouldn't have. But the `intf` expansion depends on the existence of change in #15199, which, it appears were not properly segregated